tools/skip_tests.cjs

```diff
@@ -64,6 +64,14 @@ const testsToSkip = [
   // write_text_file_test
   "writeTextFileSyncPerm", // permissions
   "writeTextFilePerm", // permissions
+
+  // mkdir, these depend on Deno.umask, which we haven't implemented yet
```

Comment:
Move these up to `// mkdir_test`

Also, add these lines to the `// timers_test` block

```JavaScript
"sleepSync", // unstable
"sleepSyncShorterPromise", // unstable
"sleepSyncLongerPromise", // unstable
```